packages/debug/browser.js

```diff
@@ -0,0 +1,5 @@
+// eslint-disable-next-line no-unused-vars
+module.exports = function debug(_name) {
+  // eslint-disable-next-line no-unused-vars
+  return function (..._args) {};
```

Comment:
Don't you want to use `console` here?

Reply:
Yeah, what stops us from doing `console.debug(name, ...args)` here?

Reply:
We could do that. I just thought that we've never really used the `debug` module in the browser and therefore we could just replace it with a no-op.

Reply:
Both would be fine for me

Reply:
It might provide value for the users. But I have no strong opinion here.

Reply:
I've implemented a simple solution based on cookies and `console.debug`.